kfs-core/src/main/resources/edu/arizona/kfs/coa/businessobject/datadictionary/OrgReviewRole.xml

```diff
@@ -41,7 +41,20 @@
       </list>
 	</property>    
   </bean>
-		
+
+  <bean id="OrgReviewRole-reviewRolesIndicator" parent="OrgReviewRole-reviewRolesIndicator-parentBean" />
+  <bean id="OrgReviewRole-reviewRolesIndicator-parentBean" abstract="true" parent="AttributeDefinition">
```

Comment:
I am wondering why you needed to completely override this attribute definition instead of just the property that you needed to change (control). Why not:
bean id="OrgReviewRole-reviewRolesIndicator" parent="OrgReviewRole-reviewRolesIndicator-parentBean"
	property name="control"
		bean parent="RadioControlDefinition"
			p:valuesFinderClass="org.kuali.kfs.coa.businessobject.options.OrgReviewRolesValuesFinder"
			p:includeKeyInLabel="true" /
	/property
/bean